src/packet/enums.rs

```diff
@@ -107,6 +109,7 @@ impl TryFrom<u8> for PacketType {
             0 => Ok(PacketType::Packet),
             1 => Ok(PacketType::Fragment),
             2 => Ok(PacketType::Heartbeat),
+            3 => Ok(PacketType::ConnectionManager),
```

Comment:
`ConnectionManager` sounds odd for a packet type. Maybe we should call this just `Connection`

Reply:
Ok, this sounds better